interface/src/app/signature/participation-type-selector.tpl.html

```diff
@@ -0,0 +1,7 @@
+<div class="alert alert-danger" role="alert" ng-if="$ctrl.errorMessage">
+  {{$ctrl.errorMessage}}
+</div>
+<select name="selectedParticipationType"
```

Comment:
Ok, changed it in my local DB and opened a PR for the change to the data migration file here: 
https://github.com/greenelab/adage-server/pull/162